### PR TITLE
UcrTextBox Enhancement

### PR DESCRIPTION
**Pull request by Patowhiz**
@africanmathsinitiative/developers this is ready for review.

### CODE REVIEW

File: ClimsoftVer4/ClimsoftVer4/ucrTextBox.vb [hunk 1]
```diff
@@ -346,21 +305,24 @@ Public Class ucrTextBox
     ''' <param name="strFieldName"></param>
     ''' <returns></returns>
     Public Overrides Function GetValue(Optional strFieldName As String = "") As Object
-        Return TextboxValue
+        Return txtBox.Text
```
Comment (maxwellfundi) [inline ClimsoftVer4/ClimsoftVer4/ucrTextBox.vb:313]: @Patowhiz  why did you change this to text from the value?

Comment (Patowhiz) [reply ClimsoftVer4/ClimsoftVer4/ucrTextBox.vb:313]: TextboxValue was a property I had originally wrote for accessing and setting the Text property for txtBox.Text . This was before we decided to use SetValue and GetValue for al controls. Now that we have those and that's the only recommended way of setting values to our controls, this property cannot now be deleted. There was a comment by Danny also suggesting this. I have removed all the references for TextboxValue property to use the GetValue and SetValue. But I'm yet to delete the property because I was surprised to see the designers using them

**Comment by Patowhiz** [comment]
@maxwellfundi are you done reviewing this. I would like to merge it so that I can use its added features
